### PR TITLE
Feature/reduce orders footprint

### PR DESCRIPTION
**Pull request by jeevanjoseph**
- Reduced orders footprint
- moved to JDK 13
- Switched to Z garbage collector
- Upgraded to SpringBoot 2.3
- Switched to Jetty in lieu of embedded Tomcat.

### CODE REVIEW

File: deploy/complete/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -135,6 +135,7 @@ services:
     image: iad.ocir.io/oracle/ateam/mushop-orders:2.1.1
     ports:
       - 8085:80
+      # - 9091:9091
```
Comment (junior) [inline deploy/complete/docker-compose/docker-compose.yml:138]: Remove comments if not using/unnecessary.

File: deploy/complete/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -152,7 +153,8 @@ services:
       - OADB_USER
       - OADB_PW
       - OADB_SERVICE=mock
-      - JAVA_OPTS=-Xms32m -Xmx256m -XX:+UseG1GC -Djava.security.egd=file:/dev/urandom -Dspring.zipkin.enabled=false -Doracle.jdbc.fanEnabled=false 
+#      - JAVA_OPTS=-Xms32m -Xmx128m -XX:MaxRAM=128m -Djava.security.egd=file:/dev/urandom -Doracle.jdbc.fanEnabled=false -XX:+UnlockExperimentalVMOptions -XX:+UseZGC -XX:+FlightRecorder -Doracle.jdbc.fanEnabled=false -Dcom.sun.management.jmxremote.rmi.port=9091 -Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.port=9091  -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=localhost
```
Comment (junior) [inline deploy/complete/docker-compose/docker-compose.yml:156]: Remove comments if not using/unnecessary.

File: src/orders/Dockerfile [hunk 3]
```diff
@@ -37,7 +37,7 @@ RUN gradle --refresh-dependencies
 FROM buildenv as appbuild
 ARG APPLICATION_NAME
 ARG VERSION
-
+ENV INSTANT_CLIENT_VERSION=${clientVersion}
```
Comment (junior) [inline src/orders/Dockerfile:40]: Where this is used?

Comment (jeevanjoseph) [reply src/orders/Dockerfile:40]: Did not stage that. Will Un stage this hunk as well. Was working on parameteizing instant client version

Comment (junior) [reply src/orders/Dockerfile:40]: Remember that the version can also affect the wallet volume

File: src/orders/src/main/resources/application.properties [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 spring.sleuth.sampler.percentage=1.0
-spring.application.name=orders
+spring.jmx.enabled=false
+
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=false
 
-management.endpoints.web.exposure.include=*
-management.endpoint.metrics.enabled=true
-management.endpoint.prometheus.enabled=true
```
Comment (junior) [inline src/orders/src/main/resources/application.properties:12]: No good metrics to prometheus?

File: src/orders/Dockerfile (hunk 3 above)
Comment (jeevanjoseph) [reply src/orders/Dockerfile]: Yes - the parametrization is not in this PR. Removed it.

File: deploy/complete/docker-compose/docker-compose.yml (hunk 2 above)
Comment (jeevanjoseph) [reply deploy/complete/docker-compose/docker-compose.yml]: done

File: deploy/complete/docker-compose/docker-compose.yml (hunk 1 above)
Comment (jeevanjoseph) [reply deploy/complete/docker-compose/docker-compose.yml]: done

**Pull request merged**